src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-01-02__create_item_status_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE item_status
+(
+    item_contribution_options_configuration_id UUID NOT NULL,
+    item_status VARCHAR(255) NOT NULL CHECK(item_status NOT IN ('Available', 'Checked out', 'Paged', 'In transit', 'Declared lost', 'Withdrawn', 'Awaiting pickup', 'Awaiting')),
```

Comment:
Is this table designed specifically for storing "Not available" statuses? If so, then it would make sense to rename it to unavailable_item_status or something like this